xml_analyzer/data_container.py

```diff
@@ -6,21 +6,31 @@ def __init__(self):
         self.mapping_element = collections.namedtuple('property_value_mapping', ['property', 'value'])
```

Comment:
declaration of a type is not very nice in __init__. Rather handle it just like a class.
MappingElement = collections.namedtuple('property_value_mapping', ['property', 'value'])

outside of the class and then you can use it as following.


map_element = MappingElement(1,2)

Reply:
Fixed with https://github.com/Squiddrone/VPParametricsSolver/pull/2/commits/6614ef2d31e5e3284ee434bbd113e446107a9606